setup

```diff
@@ -186,6 +185,7 @@ EOF
 	    /etc/systemd/system/multi-user.target.wants/bluechi-agent.service
     fi
 
+    bash -x "${INSTALLDIR}/qm-storage-settings"
```

Comment:
do we need the extra bash and the -x ?

Reply:
command takes the first argument as parameter
no exec, no output result of `/usr/share/qm/qm-rootfs`